server/builtin/debug_clean.go

```diff
@@ -37,7 +36,10 @@ func (a *builtinApp) debugClean(r *incoming.Request, creq apps.CallRequest) apps
 		Other: "Deleted all KV records.",
 	}) + "\n"
 
-	err = a.conf.StoreConfig(config.StoredConfig{}, r.Log)
+	sc := a.conf.Get().StoredConfig
```

Comment:
Because that would also wipe out the configuration settings which were introduced in this PR without setting the default value. I'm not sure that setting everything to `false` is the desired behaviour here. Maybe every setting should be set to the default value?